tools/defaults2.bzl

```diff
@@ -0,0 +1,8 @@
+load("//tools:interop.bzl", _ts_project = "ts_project")
```

Comment:
NIT: maybe a more meaningful filename? (`defaults-interop`) etc..

Reply:
Yeah, we'll rename that in a follow-up. Probably `bazel/ts_project_interop.bzl`